config/config.go

```diff
@@ -62,9 +62,12 @@ type Flags struct {
 	DisableAssetsCapture bool
 	CertValidation       bool
 
-	CloudflareStream bool
-	ElasticSearchURL string
-	ExcludedStrings  cli.StringSlice
+	CloudflareStream         bool
```

Comment:
Not your fault but I just noticed that this bool is never used, therefore completely useless, you can remove it.